16/umbraco-cms/fundamentals/backoffice/logviewer.md

```diff
@@ -30,13 +30,13 @@ Here are some example queries to help you get started. For more details on the s
 
 If you frequently use a custom query, you can save it for quick access. Type your query in the search box and click the heart icon to save it with a friendly name. Saved queries are stored in the `umbracoLogViewerQuery` table in the database.
 
-## Implementing Your Own Log Viewer
+## Implementing Your Own Log Viewer Source
 
-Umbraco allows you to implement a customn `ILogViewer` to fetch logs from alternative sources, such as **Azure Table Storage**.
+Umbraco allows you to implement a customn `ILogViewerRepository` to fetch logs from alternative sources, such as **Azure Table Storage**.
```

Comment:
Isn't it necessary to also implement a custom `ILogViewerService` too, due to the `CanViewLogs` method in `LogViewerService`?